tests/queries/0_stateless/01674_filter_by_uint8.reference

```diff
@@ -2,10 +2,10 @@
 0
 255
 1	['foo','bar']	1	1
-2	['foo','bar']	2	2
-3	['foo','bar']	3	3
-4	['foo','bar']	4	4
-5	['foo','bar']	5	5
+2	['foo','bar']	2	1
+3	['foo','bar']	3	1
+4	['foo','bar']	4	1
+5	['foo','bar']	5	1
```

Comment:
This was working only because the bug in `query_plan_optimize_prewhere`.
On master:
```
SELECT * FROM t_filter WHERE f LIMIT 5;

   ┌─s─┬─a─────────────┬─u─┬─f─┐
1. │ 1 │ ['foo','bar'] │ 1 │ 1 │
2. │ 2 │ ['foo','bar'] │ 2 │ 2 │
3. │ 3 │ ['foo','bar'] │ 3 │ 3 │
4. │ 4 │ ['foo','bar'] │ 4 │ 4 │
5. │ 5 │ ['foo','bar'] │ 5 │ 5 │
   └───┴───────────────┴───┴───┘

SELECT * FROM t_filter WHERE f LIMIT 5 settings query_plan_optimize_prewhere=0;

   ┌─s─┬─a─────────────┬─u─┬─f─┐
1. │ 1 │ ['foo','bar'] │ 1 │ 1 │
2. │ 2 │ ['foo','bar'] │ 2 │ 1 │
3. │ 3 │ ['foo','bar'] │ 3 │ 1 │
4. │ 4 │ ['foo','bar'] │ 4 │ 1 │
5. │ 5 │ ['foo','bar'] │ 5 │ 1 │
   └───┴───────────────┴───┴───┘

```